### PR TITLE
Add oimo@1.0.9 w/ npm auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #5709
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/lo-th/Oimo.js
 * Official website (optional, not the repository): https://lo-th.github.io/Oimo.js/
 * NPM package url (optional): https://www.npmjs.com/package/oimo
 * License and its reference: [MIT](https://github.com/lo-th/Oimo.js/blob/gh-pages/LICENSE)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 86
   - Count of stars: 1274
   - Count of forks: 117
 * NPM download stats (optional):
   - Downloads in the last day: 0
   - Downloads in the last week: 12
   - Downloads in the last month: 106

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@cdnjs/intern3 please help me review this PR, thanks.

Note:
I didn't add `oimo.module.js`, because 2 reasons as the followings:
1. The [examples](https://github.com/lo-th/Oimo.js/tree/gh-pages/examples) which in this library does not use it. 
2. There is no description in README say that how can we include it in HTML and how to use it. ref: https://github.com/lo-th/Oimo.js#usage

File: ajax/libs/oimo/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "author": "lo-th",
+  "license": "MIT",
+  "homepage": "https://lo-th.github.io/Oimo.js/",
+  "npmName": "oimo",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "oimo.!(module.js)"
```
Comment (extend1994) [inline ajax/libs/oimo/package.json]: `**/!(oimo.module.js)` might be a better choice because the files we need are located in `build`?

**Comment by sufuf3** [comment]
@extend1994 thanks, I modified it, please help me review this PR, again, thank you!

**Comment by sufuf3** [comment]
@hare1039 @clairetsai818  please help me review this PR, thank you!

File: ajax/libs/oimo/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "oimo",
+  "version": "1.0.9",
+  "description": "JavaScript 3D physics engine.",
+  "filename": "oimo.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/lo-th/oimo.git"
```
Comment (kennynaoh) [inline ajax/libs/oimo/package.json]: Wrong url

**Comment by sufuf3** [comment]
@kennynaoh I modified the URL, please help me review this PR, thanks.

File: ajax/libs/oimo/package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "author": "lo-th",
+  "license": "MIT",
+  "homepage": "https://lo-th.github.io/Oimo.js/",
+  "npmName": "oimo",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "**/!(oimo.module.js)"
```
Comment (kennynaoh) [inline ajax/libs/oimo/package.json]: @sufuf3 I think `**/!(oimo.module*)` is much better. What do you think?

**Comment by sufuf3** [comment]
@kennynaoh Thanks for your suggestion, I modified it, please help me review this PR, thank you,

**Pull request merged**